releases/release-1.21/release-notes/maps/pr-97395-map.yaml

```diff
@@ -1,4 +1,5 @@
 pr: 97395
 releasenote:
-  text: New Admission Controller "DenyServiceExternalIPs" is made available. Clusters
-    which do not need the service "ExternalIPs" feature should enable this controller.
+  text: New admission controller "DenyServiceExternalIPs" is available.  Clusters
```

Comment:
Should this DenyServiceExternalIPs in a `code block`

Reply:
heh good catch